firebase-appdistribution/test-app/src/main/java/com/googletest/firebase/appdistribution/testapp/MainActivity.kt

```diff
@@ -66,17 +67,15 @@ class MainActivity : AppCompatActivity() {
         signInStatus = findViewById(R.id.sign_in_status)
         progressBar = findViewById(R.id.progress_bar)
 
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
-            NotificationFeedbackTrigger.requestPermission(this)
-        }
+        firebaseAppDistribution.showFeedbackNotification(
+            R.string.terms_and_conditions, NotificationManagerCompat.IMPORTANCE_HIGH);
 
         // Set up feedback trigger menu
         feedbackTriggerMenu = findViewById(R.id.feedbackTriggerMenu)
         val items = listOf(
             FeedbackTrigger.NONE.label,
             FeedbackTrigger.SHAKE.label,
             FeedbackTrigger.SCREENSHOT.label,
-            FeedbackTrigger.NOTIFICATION.label,
```

Comment:
Why not keep this and call into the SDK for the functionality?
Or even more fancy: Have two selections for notifications: `NOTIFICATIONS_SDK` and `NOTIFICATIONS_APP` which stand for the notification provided by the SDK and (alternatively) a notification trigger implemented in the app (which can be more customized - so it's valuable example code)

Reply:
Hmm, yeah I like that idea. Done.